shared/common-adapters/input.native.js

```diff
@@ -171,7 +171,8 @@ class Input extends Component<void, Props, State> {
 
     const singlelineStyle = {
       ...commonInputStyle,
-      height: 28,
+      maxHeight: 28,
```

Comment:
:sheep: Would `_lineHeight` (defined further down) instead of 28 be appropriate here?